packages/common/src/ui/layout/tables/DataTable.tsx

```diff
@@ -69,9 +70,14 @@ export const DataTable = <T extends DomainObject>({
 
   return (
     <TableContainer
-      sx={{ display: 'flex', flexDirection: 'column', height: '100%' }}
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        height: '100%',
+        overflowX: 'unset',
```

Comment:
was getting a vertical scroll showing up - and the horizontal scroll for the table was showing below the pagination. with this it shows at the bottom of the page which was neater looking - not fussed if you want to change back tho